tests/dCommonTests/dEnumsTests/MagicEnumTests.cpp

```diff
@@ -66,48 +69,39 @@ TEST(MagicEnumTest, eWorldMessageTypeTest) {
 	LOG("Time: %lld", std::chrono::duration_cast<std::chrono::microseconds>(end - begin).count());
 
 	delete Game::logger;
-
-	#undef log_test
-	#undef log_test_invalid
 }
 
 // Test Game Message Enum Reflection
 TEST(MagicEnumTest, eGameMessageTypeTest) {
-	#define log_test(y, z)\
-		LOG("%s %s", StringifiedEnum::ToString(static_cast<eGameMessageType>(y)).data(), #z);\
-		ASSERT_STREQ(StringifiedEnum::ToString(static_cast<eGameMessageType>(y)).data(), #z);
-
-	#define log_test_invalid(y)\
-		LOG("%s", StringifiedEnum::ToString(static_cast<eGameMessageType>(y)).data());\
-		ASSERT_STRNE(StringifiedEnum::ToString(static_cast<eGameMessageType>(y)).data(), NULL);
 	
 	Game::logger = new Logger("./MagicEnumTest.log", true, true);
 	
 	// Only doing the first and last 10 for the sake of my sanity
-	log_test(0, GET_POSITION);
-	log_test(1, GET_ROTATION);
-	log_test(2, GET_LINEAR_VELOCITY);
-	log_test(3, GET_ANGULAR_VELOCITY);
-	log_test(4, GET_FORWARD_VELOCITY);
-	log_test(5, GET_PLAYER_FORWARD);
-	log_test(6, GET_FORWARD_VECTOR);
-	log_test(7, SET_POSITION);
-	log_test(8, SET_LOCAL_POSITION);
-	log_test(9, SET_ROTATION);
-	log_test(10, SET_LINEAR_VELOCITY);
-	log_test(1762, USE_SKILL_SET);
-	log_test(1763, SET_SKILL_SET_POSSESSOR);
-	log_test(1764, POPULATE_ACTION_BAR);
-	log_test(1765, GET_COMPONENT_TEMPLATE_ID);
-	log_test(1766, GET_POSSESSABLE_SKILL_SET);
-	log_test(1767, MARK_INVENTORY_ITEM_AS_ACTIVE);
-	log_test(1768, UPDATE_FORGED_ITEM);
-	log_test(1769, CAN_ITEMS_BE_REFORGED);
-	log_test(1771, NOTIFY_CLIENT_RAIL_START_FAILED);
-	log_test(1772, GET_IS_ON_RAIL);
-	log_test_invalid(32);
-	log_test_invalid(1776);
+	log_test(eGameMessageType, 0, GET_POSITION);
+	log_test(eGameMessageType, 1, GET_ROTATION);
+	log_test(eGameMessageType, 2, GET_LINEAR_VELOCITY);
+	log_test(eGameMessageType, 3, GET_ANGULAR_VELOCITY);
+	log_test(eGameMessageType, 4, GET_FORWARD_VELOCITY);
+	log_test(eGameMessageType, 5, GET_PLAYER_FORWARD);
+	log_test(eGameMessageType, 6, GET_FORWARD_VECTOR);
+	log_test(eGameMessageType, 7, SET_POSITION);
+	log_test(eGameMessageType, 8, SET_LOCAL_POSITION);
+	log_test(eGameMessageType, 9, SET_ROTATION);
+	log_test(eGameMessageType, 10, SET_LINEAR_VELOCITY);
+	log_test(eGameMessageType, 1762, USE_SKILL_SET);
+	log_test(eGameMessageType, 1763, SET_SKILL_SET_POSSESSOR);
+	log_test(eGameMessageType, 1764, POPULATE_ACTION_BAR);
+	log_test(eGameMessageType, 1765, GET_COMPONENT_TEMPLATE_ID);
+	log_test(eGameMessageType, 1766, GET_POSSESSABLE_SKILL_SET);
+	log_test(eGameMessageType, 1767, MARK_INVENTORY_ITEM_AS_ACTIVE);
+	log_test(eGameMessageType, 1768, UPDATE_FORGED_ITEM);
+	log_test(eGameMessageType, 1769, CAN_ITEMS_BE_REFORGED);
+	log_test(eGameMessageType, 1771, NOTIFY_CLIENT_RAIL_START_FAILED);
+	log_test(eGameMessageType, 1772, GET_IS_ON_RAIL);
+	log_test_invalid(eGameMessageType, 32);
+	log_test_invalid(eGameMessageType, 1776);
 
+	srand(time(NULL));
 	auto begin = std::chrono::high_resolution_clock::now();
```

Comment:
I also noticed the timing code is repeated, could that be simplified down to one call as well?